### ISSUE TITLE
Wireshark segmentation fault in libQt6Widgets.so.6.8.0

### ISSUE THREAD

**Issue by JustScreaMy** [NixOS/nixpkgs#356019]
## Describe the bug
Whenever I'm doing anything, like analyzing packets and clicking around, two or more doubleclicks (showing detailed information) on any packets crashes the program

## Steps To Reproduce

1. Open Wireshark
2. Start inspecting packets with double-click
3. Repeat step 2 until crashes

## Expected behavior

No crash

## Additional context
Logs when running with sudo from live capture
`zsh: segmentation fault  sudo wireshark`

```
lis 14 23:22:02 wenar-nix kernel: .wireshark-wrap[53445]: segfault at 8 ip 00007fc18cbe8ae4 sp 00007ffcc4b01ed8 error 4 in libQt6Widgets.so.6.8.0[1e8ae4,7fc18cb84000+487000] likely on CPU 7 (core 1, socket 0)
lis 14 23:22:02 wenar-nix kernel: Code: 25 28 00 00 00 75 0c 48 83 c4 30 5b 31 c0 31 f6 31 ff c3 e8 be 58 fa ff 66 66 2e 0f 1f 84 00 00 00 00 00 0f 1f 00 f3 0f 1e fa <48> 8b 4e 08 48 89 f8 f3 0f 6f 81 38 01 00 00 48 8b 91 38 01 00 00
lis 14 23:22:02 wenar-nix systemd-coredump[53527]: Process 53445 (.wireshark-wrap) of user 0 terminated abnormally with signal 11/SEGV, processing...
lis 14 23:22:02 wenar-nix systemd[1]: Started Process Core Dump (PID 53527/UID 0).
lis 14 23:22:02 wenar-nix systemd[1]: Started Pass systemd-coredump journal entries to relevant user for potential DrKonqi handling.
lis 14 23:22:02 wenar-nix systemd-coredump[53528]: Resource limits disable core dumping for process 53445 (.wireshark-wrap).
lis 14 23:22:02 wenar-nix systemd-coredump[53528]: [🡕] Process 53445 (.wireshark-wrap) of user 0 terminated abnormally without generating a coredump.
lis 14 23:22:02 wenar-nix systemd[1]: systemd-coredump@16-53527-0.service: Deactivated successfully.
lis 14 23:22:02 wenar-nix drkonqi-coredump-processor[53529]: Entry doesn't look like a dump. This may have been a vaccum run. Nothing to process.
lis 14 23:22:02 wenar-nix drkonqi-coredump-processor[53529]: "/nix/store/7qmisxv7555iphq1xbm3r3796s2g8czq-wireshark-qt-4.2.8/bin/.wireshark-wrapped" 53445 ""
lis 14 23:22:02 wenar-nix drkonqi-coredump-processor[53529]: The socket path doesn't exist @ "/run/user/0/drkonqi-coredump-launcher"
```

Logs when running without sudo from capture file
`zsh: segmentation fault (core dumped) `
```
lis 14 23:24:45 wenar-nix kernel: .wireshark-wrap[53697]: segfault at 8 ip 00007ff8407e8ae4 sp 00007ffd65066858 error 4 in libQt6Widgets.so.6.8.0[1e8ae4,7ff840784000+487000] likely on CPU 4 (core 4, socket 0)
lis 14 23:24:45 wenar-nix kernel: Code: 25 28 00 00 00 75 0c 48 83 c4 30 5b 31 c0 31 f6 31 ff c3 e8 be 58 fa ff 66 66 2e 0f 1f 84 00 00 00 00 00 0f 1f 00 f3 0f 1e fa <48> 8b 4e 08 48 89 f8 f3 0f 6f 81 38 01 00 00 48 8b 91 38 01 00 00
lis 14 23:24:45 wenar-nix systemd-coredump[53773]: Process 53697 (.wireshark-wrap) of user 1000 terminated abnormally with signal 11/SEGV, processing...
lis 14 23:24:45 wenar-nix systemd[1]: Started Process Core Dump (PID 53773/UID 0).
lis 14 23:24:45 wenar-nix systemd[1]: Started Pass systemd-coredump journal entries to relevant user for potential DrKonqi handling.
lis 14 23:24:45 wenar-nix systemd-timesyncd[841]: Network configuration changed, trying to establish connection.
lis 14 23:24:45 wenar-nix systemd-timesyncd[841]: Network configuration changed, trying to establish connection.
lis 14 23:24:45 wenar-nix systemd-timesyncd[841]: Network configuration changed, trying to establish connection.
lis 14 23:24:45 wenar-nix systemd-timesyncd[841]: Network configuration changed, trying to establish connection.
lis 14 23:24:45 wenar-nix systemd-timesyncd[841]: Network configuration changed, trying to establish connection.
lis 14 23:24:45 wenar-nix systemd-timesyncd[841]: Network configuration changed, trying to establish connection.
lis 14 23:24:45 wenar-nix systemd-timesyncd[841]: Contacted time server 194.182.77.109:123 (0.nixos.pool.ntp.org).
lis 14 23:24:45 wenar-nix systemd-coredump[53774]: [🡕] Process 53697 (.wireshark-wrap) of user 1000 dumped core.

                                                   Module libsbc.so.1 without build-id.
                                                   Module sbc.so without build-id.
                                                   Module opus_dec.so without build-id.
                                                   Module l16mono.so without build-id.
                                                   Module libbcg729.so.0 without build-id.
                                                   Module g729.so without build-id.
                                                   Module g726.so without build-id.
                                                   Module libLerc.so.4 without build-id.
                                                   Module libwebp.so.7 without build-id.
                                                   Module libtiff.so.6 without build-id.
                                                   Module libspandsp.so.3 without build-id.
                                                   Module g722.so without build-id.
                                                   Module g711.so without build-id.
                                                   Module libopencore-amrnb.so.0 without build-id.
                                                   Module amrnb.so without build-id.
                                                   Module wimaxmacphy.so without build-id.
                                                   Module wimaxasncp.so without build-id.
                                                   Module wimax.so without build-id.
                                                   Module unistim.so without build-id.
                                                   Module transum.so without build-id.
                                                   Module stats_tree.so without build-id.
                                                   Module profinet.so without build-id.
                                                   Module opcua.so without build-id.
                                                   Module mate.so without build-id.
                                                   Module irda.so without build-id.
                                                   Module gryphon.so without build-id.
                                                   Module ethercat.so without build-id.
                                                   Module usbdump.so without build-id.
                                                   Module liblcms2.so.2 without build-id.
                                                   Module libraw.so.23 without build-id.
                                                   Module libbrotlienc.so.1 without build-id.
                                                   Module libjxl_cms.so.0.11 without build-id.
                                                   Module libjxl_threads.so.0.11 without build-id.
                                                   Module libjxl.so.0.11 without build-id.
                                                   Module libatomic.so.1 without build-id.
                                                   Module libnuma.so.1 without build-id.
                                                   Module libsharpyuv.so.0 without build-id.
                                                   Module libde265.so without build-id.
                                                   Module libx265.so.209 without build-id.
                                                   Module libheif.so.1 without build-id.
                                                   Module libdeflate.so.0 without build-id.
                                                   Module libImath-3_1.so.29 without build-id.
                                                   Module libOpenEXRCore-3_2.so.31 without build-id.
                                                   Module libIex-3_2.so.31 without build-id.
                                                   Module libIlmThread-3_2.so.31 without build-id.
                                                   Module libOpenEXR-3_2.so.31 without build-id.
                                                   Module libvmaf.so.3 without build-id.
                                                   Module libaom.so.3 without build-id.
                                                   Module libdav1d.so.7 without build-id.
                                                   Module libyuv.so without build-id.
                                                   Module libavif.so.16 without build-id.
                                                   Module libjpeg.so.62 without build-id.
                                                   Module libxcb-shm.so.0 without build-id.
                                                   Module libpciaccess.so.0 without build-id.
                                                   Module libxcb-dri2.so.0 without build-id.
                                                   Module libdrm_intel.so.1 without build-id.
                                                   Module libdrm_amdgpu.so.1 without build-id.
                                                   Module libdrm_radeon.so.1 without build-id.
                                                   Module libsensors.so.5 without build-id.
                                                   Module libxshmfence.so.1 without build-id.
                                                   Module libxcb-sync.so.1 without build-id.
                                                   Module libxcb-xfixes.so.0 without build-id.
                                                   Module libxcb-present.so.0 without build-id.
                                                   Module libxcb-dri3.so.0 without build-id.
                                                   Module libX11-xcb.so.1 without build-id.
                                                   Module libxcb-randr.so.0 without build-id.
                                                   Module libdrm.so.2 without build-id.
                                                   Module liblzma.so.5 without build-id.
                                                   Module libimobiledevice-glue-1.0.so.0 without build-id.
                                                   Module libusbmuxd-2.0.so.7 without build-id.
                                                   Module libltdl.so.7 without build-id.
                                                   Module libvorbisfile.so.3 without build-id.
                                                   Module libplist-2.0.so.4 without build-id.
                                                   Module libimobiledevice-1.0.so.6 without build-id.
                                                   Module libcanberra.so.0 without build-id.
                                                   Module libXrender.so.1 without build-id.
                                                   Module libxcb-keysyms.so.1 without build-id.
                                                   Module libXfixes.so.3 without build-id.
                                                   Module libxcb-res.so.0 without build-id.
                                                   Module libattr.so.1 without build-id.
                                                   Module libacl.so.1 without build-id.
                                                   Module libXcursor.so.1 without build-id.
                                                   Module libselinux.so.1 without build-id.
                                                   Module libpsl.so.5 without build-id.
                                                   Module libssh2.so.1 without build-id.
                                                   Module libmp3lame.so.0 without build-id.
                                                   Module libmpg123.so.0 without build-id.
                                                   Module libopus.so.0 without build-id.
                                                   Module libvorbisenc.so.2 without build-id.
                                                   Module libvorbis.so.0 without build-id.
                                                   Module libFLAC.so.12 without build-id.
                                                   Module libduktape.so.207 without build-id.
                                                   Module libXdmcp.so.6 without build-id.
                                                   Module libXau.so.6 without build-id.
                                                   Module libsndfile.so.1 without build-id.
                                                   Module libffi.so.8 without build-id.
                                                   Module libpxbackend-1.0.so without build-id.
                                                   Module libncursesw.so.6 without build-id.
                                                   Module libreadline.so.8 without build-id.
                                                   Module libkeyutils.so.1 without build-id.
                                                   Module libkrb5support.so.0 without build-id.
                                                   Module libcom_err.so.3 without build-id.
                                                   Module libgmp.so.10 without build-id.
                                                   Module libhogweed.so.6 without build-id.
                                                   Module libnettle.so.8 without build-id.
                                                   Module libtasn1.so.6 without build-id.
                                                   Module libunistring.so.5 without build-id.
                                                   Module libidn2.so.0 without build-id.
                                                   Module libp11-kit.so.0 without build-id.
                                                   Module libbrotlicommon.so.1 without build-id.
                                                   Module libgomp.so.1 without build-id.
                                                   Module libcap.so.2 without build-id.
                                                   Module libgraphite2.so.3 without build-id.
                                                   Module libxcb.so.1 without build-id.
                                                   Module libexpat.so.1 without build-id.
                                                   Module libbz2.so.1 without build-id.
                                                   Module libpulsecommon-17.0.so without build-id.
                                                   Module libproxy.so.1 without build-id.
                                                   Module libgssapi_krb5.so.2 without build-id.
                                                   Module libGLdispatch.so.0 without build-id.
                                                   Module libXext.so.6 without build-id.
                                                   Module libgpg-error.so.0 without build-id.
                                                   Module libpcre2-8.so.0 without build-id.
                                                   Module libsnappy.so.1 without build-id.
                                                   Module libsmi.so.2 without build-id.
                                                   Module libnghttp3.so.9 without build-id.
                                                   Module libnghttp2.so.14 without build-id.
                                                   Module liblz4.so.1 without build-id.
                                                   Module liblua.so.5.2 without build-id.
                                                   Module libxml2.so.2 without build-id.
                                                   Module libk5crypto.so.3 without build-id.
                                                   Module libkrb5.so.3 without build-id.
                                                   Module libgnutls.so.30 without build-id.
                                                   Module libcares.so.2 without build-id.
                                                   Module libbrotlidec.so.1 without build-id.
                                                   Module libzstd.so.1 without build-id.
                                                   Module libpcre2-16.so.0 without build-id.
                                                   Module libb2.so.1 without build-id.
                                                   Module libdouble-conversion.so.3 without build-id.
                                                   Module libfreetype.so.6 without build-id.
                                                   Module libmd4c.so.0 without build-id.
                                                   Module libharfbuzz.so.0 without build-id.
                                                   Module libpng16.so.16 without build-id.
                                                   Module libX11.so.6 without build-id.
                                                   Module libfontconfig.so.1 without build-id.
                                                   Module libEGL.so.1 without build-id.
                                                   Module libpulse.so.0 without build-id.
                                                   Module libxkbcommon.so.0 without build-id.
                                                   Module libOpenGL.so.0 without build-id.
                                                   Module libGLX.so.0 without build-id.
                                                   Module libicudata.so.74 without build-id.
                                                   Module libicuuc.so.74 without build-id.
                                                   Module libicui18n.so.74 without build-id.
                                                   Module libz.so.1 without build-id.
                                                   Module libfftw3f.so.3 without build-id.
                                                   Module libgcc_s.so.1 without build-id.
                                                   Module libstdc++.so.6 without build-id.
                                                   Module libgcrypt.so.20 without build-id.
                                                   Module libwsutil.so.15 without build-id.
                                                   Module libwiretap.so.14 without build-id.
                                                   Module libwireshark.so.17 without build-id.
                                                   Module libnl-3.so.200 without build-id.
                                                   Module libnl-route-3.so.200 without build-id.
                                                   Module libnl-genl-3.so.200 without build-id.
                                                   Module libpcap.so.1 without build-id.
                                                   Module libminizip.so.1 without build-id.
                                                   Module libspeexdsp.so.1 without build-id.
                                                   Module .wireshark-wrapped without build-id.
                                                   Stack trace of thread 53697:
                                                   #0  0x00007ff8407e8ae4 _ZNK7QWidget14accessibleNameEv (libQt6Widgets.so.6 + 0x1e8ae4)
                                                   #1  0x00007ff840a6c3cf _ZNK17QAccessibleWidget4textEN11QAccessible4TextE (libQt6Widgets.so.6 + 0x46c3cf)
                                                   #2  0x00007ff8407f2aa0 _ZN7QWidget14setWindowTitleERK7QString (libQt6Widgets.so.6 + 0x1f2aa0)
                                                   #3  0x000055b7a029c0c4 _ZN15WiresharkDialog17setWindowSubtitleERK7QString (.wireshark-wrapped + 0x40f0c4)
                                                   #4  0x000055b7a029c3e5 _ZN15WiresharkDialogC1ER7QWidgetR11CaptureFile (.wireshark-wrapped + 0x40f3e5)
                                                   #5  0x000055b7a01a7101 _ZN12PacketDialogC1ER7QWidgetR11CaptureFileP11_frame_data (.wireshark-wrapped + 0x31a101)
                                                   #6  0x000055b7a02e334f _ZN19WiresharkMainWindow16openPacketDialogEb (.wireshark-wrapped + 0x45634f)
                                                   #7  0x00007ff83f3f9bdc _Z10doActivateILb0EEvP7QObjectiPPv (libQt6Core.so.6 + 0x1f9bdc)
                                                   #8  0x00007ff840a85569 _ZN17QAbstractItemView13doubleClickedERK11QModelIndex (libQt6Widgets.so.6 + 0x485569)
                                                   #9  0x00007ff840b20216 _ZN9QTreeView21mouseDoubleClickEventEP11QMouseEvent (libQt6Widgets.so.6 + 0x520216)
                                                   #10 0x00007ff840802e0b _ZN7QWidget5eventEP6QEvent (libQt6Widgets.so.6 + 0x202e0b)
                                                   #11 0x00007ff840895d16 _ZN6QFrame5eventEP6QEvent (libQt6Widgets.so.6 + 0x295d16)
                                                   #12 0x00007ff83f39664e _ZN23QCoreApplicationPrivate29sendThroughObjectEventFiltersEP7QObjectP6QEvent (libQt6Core.so.6 + 0x19664e)
                                                   #13 0x00007ff8407a0739 _ZN19QApplicationPrivate13notify_helperEP7QObjectP6QEvent (libQt6Widgets.so.6 + 0x1a0739)
                                                   #14 0x00007ff8407a9ea2 _ZN12QApplication6notifyEP7QObjectP6QEvent (libQt6Widgets.so.6 + 0x1a9ea2)
                                                   #15 0x00007ff83f396948 _ZN16QCoreApplication15notifyInternal2EP7QObjectP6QEvent (libQt6Core.so.6 + 0x196948)
                                                   #16 0x00007ff8407a8ae8 _ZN19QApplicationPrivate14sendMouseEventEP7QWidgetP11QMouseEventS1_S1_PS1_R8QPointerIS0_Ebb (libQt6Widgets.so.6 + 0x1a8ae8)
                                                   #17 0x00007ff840816048 _ZN13QWidgetWindow16handleMouseEventEP11QMouseEvent (libQt6Widgets.so.6 + 0x216048)
                                                   #18 0x00007ff840818670 _ZN13QWidgetWindow5eventEP6QEvent (libQt6Widgets.so.6 + 0x218670)
                                                   #19 0x00007ff8407a0749 _ZN19QApplicationPrivate13notify_helperEP7QObjectP6QEvent (libQt6Widgets.so.6 + 0x1a0749)
                                                   #20 0x00007ff83f396948 _ZN16QCoreApplication15notifyInternal2EP7QObjectP6QEvent (libQt6Core.so.6 + 0x196948)
                                                   #21 0x00007ff83fbf0baf _ZN22QGuiApplicationPrivate17processMouseEventEPN29QWindowSystemInterfacePrivate10MouseEventE (libQt6Gui.so.6 + 0x1f0baf)
                                                   #22 0x00007ff83fc558d4 _ZN22QWindowSystemInterface22sendWindowSystemEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE (libQt6Gui.so.6 + 0x2558d4)
                                                   #23 0x00007ff84014ae24 _ZL23userEventSourceDispatchP8_GSourcePFiPvES1_ (libQt6Gui.so.6 + 0x74ae24)
                                                   #24 0x00007ff83650b571 g_main_context_dispatch_unlocked (libglib-2.0.so.0 + 0x60571)
                                                   #25 0x00007ff83650d6b0 g_main_context_iterate_unlocked.isra.0 (libglib-2.0.so.0 + 0x626b0)
                                                   #26 0x00007ff83650de9c g_main_context_iteration (libglib-2.0.so.0 + 0x62e9c)
                                                   #27 0x00007ff83f674f30 _ZN20QEventDispatcherGlib13processEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE (libQt6Core.so.6 + 0x474f30)
                                                   #28 0x00007ff83f3a39fa _ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE (libQt6Core.so.6 + 0x1a39fa)
                                                   #29 0x00007ff83f39f166 _ZN16QCoreApplication4execEv (libQt6Core.so.6 + 0x19f166)
                                                   #30 0x000055b79ffdd89f main (.wireshark-wrapped + 0x15089f)
                                                   #31 0x00007ff835e3127e __libc_start_call_main (libc.so.6 + 0x2a27e)
                                                   #32 0x00007ff835e31339 __libc_start_main@@GLIBC_2.34 (libc.so.6 + 0x2a339)
                                                   #33 0x000055b79ffe06a5 _start (.wireshark-wrapped + 0x1536a5)

                                                   Stack trace of thread 53708:
                                                   #0  0x00007ff835e9437e __futex_abstimed_wait_common (libc.so.6 + 0x8d37e)
                                                   #1  0x00007ff835e971f5 pthread_cond_timedwait@@GLIBC_2.3.2 (libc.so.6 + 0x901f5)
                                                   #2  0x00007ff83f55bc92 _ZN14QWaitCondition4waitEP6QMutex14QDeadlineTimer (libQt6Core.so.6 + 0x35bc92)
                                                   #3  0x00007ff83f5586fd _ZN17QThreadPoolThread3runEv (libQt6Core.so.6 + 0x3586fd)
                                                   #4  0x00007ff83f54eae4 _ZN14QThreadPrivate5startEPv (libQt6Core.so.6 + 0x34eae4)
                                                   #5  0x00007ff835e97d02 start_thread (libc.so.6 + 0x90d02)
                                                   #6  0x00007ff835f173ac __clone3 (libc.so.6 + 0x1103ac)

                                                   Stack trace of thread 53698:
                                                   #0  0x00007ff835f096b6 ppoll (libc.so.6 + 0x1026b6)
                                                   #1  0x00007ff83650d763 g_main_context_iterate_unlocked.isra.0 (libglib-2.0.so.0 + 0x62763)
                                                   #2  0x00007ff83650de9c g_main_context_iteration (libglib-2.0.so.0 + 0x62e9c)
                                                   #3  0x00007ff83f674f30 _ZN20QEventDispatcherGlib13processEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE (libQt6Core.so.6 + 0x474f30)
                                                   #4  0x00007ff83f3a39fa _ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE (libQt6Core.so.6 + 0x1a39fa)
                                                   #5  0x00007ff83f4b1070 _ZN7QThread4execEv (libQt6Core.so.6 + 0x2b1070)
                                                   #6  0x00007ff83336cb2e _ZN22QDBusConnectionManager3runEv (libQt6DBus.so.6 + 0x40b2e)
                                                   #7  0x00007ff83f54eae4 _ZN14QThreadPrivate5startEPv (libQt6Core.so.6 + 0x34eae4)
                                                   #8  0x00007ff835e97d02 start_thread (libc.so.6 + 0x90d02)
                                                   #9  0x00007ff835f173ac __clone3 (libc.so.6 + 0x1103ac)

                                                   Stack trace of thread 53707:
                                                   #0  0x00007ff835e9437e __futex_abstimed_wait_common (libc.so.6 + 0x8d37e)
                                                   #1  0x00007ff835e971f5 pthread_cond_timedwait@@GLIBC_2.3.2 (libc.so.6 + 0x901f5)
                                                   #2  0x00007ff83f55bc92 _ZN14QWaitCondition4waitEP6QMutex14QDeadlineTimer (libQt6Core.so.6 + 0x35bc92)
                                                   #3  0x00007ff83f5586fd _ZN17QThreadPoolThread3runEv (libQt6Core.so.6 + 0x3586fd)
                                                   #4  0x00007ff83f54eae4 _ZN14QThreadPrivate5startEPv (libQt6Core.so.6 + 0x34eae4)
                                                   #5  0x00007ff835e97d02 start_thread (libc.so.6 + 0x90d02)
                                                   #6  0x00007ff835f173ac __clone3 (libc.so.6 + 0x1103ac)

                                                   Stack trace of thread 53699:
                                                   #0  0x00007ff835f090af __poll (libc.so.6 + 0x1020af)
                                                   #1  0x00007ff83077b591 _ZN15QtWaylandClient11EventThread3runEv (libQt6WaylandClient.so.6 + 0x87591)
                                                   #2  0x00007ff83f54eae4 _ZN14QThreadPrivate5startEPv (libQt6Core.so.6 + 0x34eae4)
                                                   #3  0x00007ff835e97d02 start_thread (libc.so.6 + 0x90d02)
                                                   #4  0x00007ff835f173ac __clone3 (libc.so.6 + 0x1103ac)

                                                   Stack trace of thread 53715:
                                                   #0  0x00007ff835f1513d syscall (libc.so.6 + 0x10e13d)
                                                   #1  0x00007ff83653dc00 g_cond_wait_until (libglib-2.0.so.0 + 0x92c00)
                                                   #2  0x00007ff8364d46db g_async_queue_pop_intern_unlocked (libglib-2.0.so.0 + 0x296db)
                                                   #3  0x00007ff8364d4d51 g_async_queue_timeout_pop (libglib-2.0.so.0 + 0x29d51)
                                                   #4  0x00007ff83653e375 g_thread_pool_thread_proxy (libglib-2.0.so.0 + 0x93375)
                                                   #5  0x00007ff83653debe g_thread_proxy (libglib-2.0.so.0 + 0x92ebe)
                                                   #6  0x00007ff835e97d02 start_thread (libc.so.6 + 0x90d02)
                                                   #7  0x00007ff835f173ac __clone3 (libc.so.6 + 0x1103ac)

                                                   Stack trace of thread 53710:
                                                   #0  0x00007ff835e9437e __futex_abstimed_wait_common (libc.so.6 + 0x8d37e)
                                                   #1  0x00007ff835e971f5 pthread_cond_timedwait@@GLIBC_2.3.2 (libc.so.6 + 0x901f5)
                                                   #2  0x00007ff83f55bc92 _ZN14QWaitCondition4waitEP6QMutex14QDeadlineTimer (libQt6Core.so.6 + 0x35bc92)
                                                   #3  0x00007ff83f5586fd _ZN17QThreadPoolThread3runEv (libQt6Core.so.6 + 0x3586fd)
                                                   #4  0x00007ff83f54eae4 _ZN14QThreadPrivate5startEPv (libQt6Core.so.6 + 0x34eae4)
                                                   #5  0x00007ff835e97d02 start_thread (libc.so.6 + 0x90d02)
                                                   #6  0x00007ff835f173ac __clone3 (libc.so.6 + 0x1103ac)

                                                   Stack trace of thread 53704:
                                                   #0  0x00007ff835f096b6 ppoll (libc.so.6 + 0x1026b6)
                                                   #1  0x00007ff83650d763 g_main_context_iterate_unlocked.isra.0 (libglib-2.0.so.0 + 0x62763)
                                                   #2  0x00007ff83650de9c g_main_context_iteration (libglib-2.0.so.0 + 0x62e9c)
                                                   #3  0x00007ff83f674f30 _ZN20QEventDispatcherGlib13processEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE (libQt6Core.so.6 + 0x474f30)
                                                   #4  0x00007ff83f3a39fa _ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE (libQt6Core.so.6 + 0x1a39fa)
                                                   #5  0x00007ff83f4b1070 _ZN7QThread4execEv (libQt6Core.so.6 + 0x2b1070)
                                                   #6  0x00007ff83f54eae4 _ZN14QThreadPrivate5startEPv (libQt6Core.so.6 + 0x34eae4)
                                                   #7  0x00007ff835e97d02 start_thread (libc.so.6 + 0x90d02)
                                                   #8  0x00007ff835f173ac __clone3 (libc.so.6 + 0x1103ac)

                                                   Stack trace of thread 53706:
                                                   #0  0x00007ff835e9437e __futex_abstimed_wait_common (libc.so.6 + 0x8d37e)
                                                   #1  0x00007ff835e971f5 pthread_cond_timedwait@@GLIBC_2.3.2 (libc.so.6 + 0x901f5)
                                                   #2  0x00007ff83f55bc92 _ZN14QWaitCondition4waitEP6QMutex14QDeadlineTimer (libQt6Core.so.6 + 0x35bc92)
                                                   #3  0x00007ff83f5586fd _ZN17QThreadPoolThread3runEv (libQt6Core.so.6 + 0x3586fd)
                                                   #4  0x00007ff83f54eae4 _ZN14QThreadPrivate5startEPv (libQt6Core.so.6 + 0x34eae4)
                                                   #5  0x00007ff835e97d02 start_thread (libc.so.6 + 0x90d02)
                                                   #6  0x00007ff835f173ac __clone3 (libc.so.6 + 0x1103ac)

                                                   Stack trace of thread 53702:
                                                   #0  0x00007ff835e9437e __futex_abstimed_wait_common (libc.so.6 + 0x8d37e)
                                                   #1  0x00007ff835e971f5 pthread_cond_timedwait@@GLIBC_2.3.2 (libc.so.6 + 0x901f5)
                                                   #2  0x00007ff83f55bc92 _ZN14QWaitCondition4waitEP6QMutex14QDeadlineTimer (libQt6Core.so.6 + 0x35bc92)
                                                   #3  0x00007ff83f5586fd _ZN17QThreadPoolThread3runEv (libQt6Core.so.6 + 0x3586fd)
                                                   #4  0x00007ff83f54eae4 _ZN14QThreadPrivate5startEPv (libQt6Core.so.6 + 0x34eae4)
                                                   #5  0x00007ff835e97d02 start_thread (libc.so.6 + 0x90d02)
                                                   #6  0x00007ff835f173ac __clone3 (libc.so.6 + 0x1103ac)

                                                   Stack trace of thread 53709:
                                                   #0  0x00007ff835e9437e __futex_abstimed_wait_common (libc.so.6 + 0x8d37e)
                                                   #1  0x00007ff835e971f5 pthread_cond_timedwait@@GLIBC_2.3.2 (libc.so.6 + 0x901f5)
                                                   #2  0x00007ff83f55bc92 _ZN14QWaitCondition4waitEP6QMutex14QDeadlineTimer (libQt6Core.so.6 + 0x35bc92)
                                                   #3  0x00007ff83f5586fd _ZN17QThreadPoolThread3runEv (libQt6Core.so.6 + 0x3586fd)
                                                   #4  0x00007ff83f54eae4 _ZN14QThreadPrivate5startEPv (libQt6Core.so.6 + 0x34eae4)
                                                   #5  0x00007ff835e97d02 start_thread (libc.so.6 + 0x90d02)
                                                   #6  0x00007ff835f173ac __clone3 (libc.so.6 + 0x1103ac)

                                                   Stack trace of thread 53700:
                                                   #0  0x00007ff835f090af __poll (libc.so.6 + 0x1020af)
                                                   #1  0x00007ff83077b591 _ZN15QtWaylandClient11EventThread3runEv (libQt6WaylandClient.so.6 + 0x87591)
                                                   #2  0x00007ff83f54eae4 _ZN14QThreadPrivate5startEPv (libQt6Core.so.6 + 0x34eae4)
                                                   #3  0x00007ff835e97d02 start_thread (libc.so.6 + 0x90d02)
                                                   #4  0x00007ff835f173ac __clone3 (libc.so.6 + 0x1103ac)

                                                   Stack trace of thread 53714:
                                                   #0  0x00007ff835f1513d syscall (libc.so.6 + 0x10e13d)
                                                   #1  0x00007ff83653d760 g_cond_wait (libglib-2.0.so.0 + 0x92760)
                                                   #2  0x00007ff8364d470b g_async_queue_pop_intern_unlocked (libglib-2.0.so.0 + 0x2970b)
                                                   #3  0x00007ff83653e1b2 g_thread_pool_spawn_thread (libglib-2.0.so.0 + 0x931b2)
                                                   #4  0x00007ff83653debe g_thread_proxy (libglib-2.0.so.0 + 0x92ebe)
                                                   #5  0x00007ff835e97d02 start_thread (libc.so.6 + 0x90d02)
                                                   #6  0x00007ff835f173ac __clone3 (libc.so.6 + 0x1103ac)

                                                   Stack trace of thread 53711:
                                                   #0  0x00007ff835e9437e __futex_abstimed_wait_common (libc.so.6 + 0x8d37e)
                                                   #1  0x00007ff835e971f5 pthread_cond_timedwait@@GLIBC_2.3.2 (libc.so.6 + 0x901f5)
                                                   #2  0x00007ff83f55bc92 _ZN14QWaitCondition4waitEP6QMutex14QDeadlineTimer (libQt6Core.so.6 + 0x35bc92)
                                                   #3  0x00007ff83f5586fd _ZN17QThreadPoolThread3runEv (libQt6Core.so.6 + 0x3586fd)
                                                   #4  0x00007ff83f54eae4 _ZN14QThreadPrivate5startEPv (libQt6Core.so.6 + 0x34eae4)
                                                   #5  0x00007ff835e97d02 start_thread (libc.so.6 + 0x90d02)
                                                   #6  0x00007ff835f173ac __clone3 (libc.so.6 + 0x1103ac)
                                                   ELF object binary architecture: AMD x86-64
lis 14 23:24:45 wenar-nix systemd[1]: systemd-coredump@17-53773-0.service: Deactivated successfully.
lis 14 23:24:45 wenar-nix systemd[1]: systemd-coredump@17-53773-0.service: Consumed 518ms CPU time, 332.4M memory peak, 19.3M written to disk.
lis 14 23:24:45 wenar-nix drkonqi-coredump-processor[53775]: "/nix/store/7qmisxv7555iphq1xbm3r3796s2g8czq-wireshark-qt-4.2.8/bin/.wireshark-wrapped" 53697 "/var/lib/systemd/coredump/core.\\x2ewireshark-wrap.1000.0aff9487beed48ad926946d0ecf29d39.53697.1731623084000000.zst"
lis 14 23:24:45 wenar-nix systemd[1403]: Started Launch DrKonqi for a systemd-coredump crash (PID 53775/UID 0).
lis 14 23:24:45 wenar-nix drkonqi-coredump-launcher[53794]: Unable to find file for pid 53697 expected at "kcrash-metadata/.wireshark-wrapped.0aff9487beed48ad926946d0ecf29d39.53697.ini"
lis 14 23:24:45 wenar-nix systemd[1]: drkonqi-coredump-processor@17-53773-0.service: Deactivated successfully.
lis 14 23:24:45 wenar-nix drkonqi-coredump-launcher[53794]: Nothing handled the dump :O
```

## Metadata
 - system: `"x86_64-linux"`
 - host os: `Linux 6.11.7, NixOS, 24.11 (Vicuna), 24.11.20241109.76612b1`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.10`
 - nixpkgs: `/nix/store/0sxsfzswhjck7f4sbsvznz285s4i7y64-source`

And output from KDE Plasma settings
```
Operating System: NixOS 24.11
KDE Plasma Version: 6.2.3
KDE Frameworks Version: 6.8.0
Qt Version: 6.8.0
Kernel Version: 6.11.7 (64-bit)
Graphics Platform: Wayland
Processors: 12 × 11th Gen Intel® Core™ i5-11600KF @ 3.90GHz
Memory: 15,5 GiB of RAM
Graphics Processor: NVIDIA GeForce RTX 3070 Ti/PCIe/SSE2
Manufacturer: ASUS
```

## Notify maintainers
@bjornfor @fpletz 

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by JustScreaMy** [comment]
Update: find out that it only happens on the version 4.2.8
Trying to run 4.2.5 works fine and doesnt crash

**Comment by fpletz** [comment]
May be fixed with version 4.4 in #344914.

**Comment by JustScreaMy** [comment]
Version 4.4 fixes this, thanks!
